### ISSUE TITLE
The "amount" of data augmentation

### ISSUE THREAD

**Issue by guuusss** [NVlabs/PWC-Net#64]
Dear Dr. @deqings :
Thank you for your code.I have reimplemented the PWC-Net with tensorflow.In my code,I find that the net converge so slowly if augment on all of the training sets,say:FlyingChairs.The types of data augmentation are followed FlowNet2:
1.Translate **all of images** by a value between **-20 and +20** percent of original size on x- and y-axis independently
2.Scale **all of images** by a factor between **90 and 200** percent of original size
3.**No horizontal flip and vertically flip is used**
4.Add the Gaussian noise that has a sigma uniformly sampled from [0, 0.04]
5.Add the contrast sampled within [−0.8, 0.4]
6.Multiplicative color changes to the RGB channels per image from [0.5, 2]
7.gamma values from [0.7, 1.5] and additive brightness changes using Gaussian with a sigma of 0.2.
Should I not do data enhancements on all data？I would be very appreciated if you can help me:)
Thanks and regards.

**Comment by xianshunw** [comment]
@guuusss when you remove the data augmentation, will the generalization ability of your network deteriorate?

**Comment by guuusss** [comment]
My network will converge fast on FlyingChairs when I remove the data augmentation.

**Comment by xianshunw** [comment]
@ Thanks for your replaying!! I find that when I remove the data augmentation, the network does converge better on Chairs but get the generalization ability worse on sintel.

**Comment by xianshunw** [comment]
@guuusss do you still working on flow? We have two guys here following PWC-Net. Would you be interested in joining our discussion group?

**Comment by guuusss** [comment]
@xianshunw Sorry for my late reply, It is a great pleasure for me to join in your group. So, how can i take part in it?

**Comment by xianshunw** [comment]
@guuusss perfect!! we have a wechat group here,
![image](https://user-images.githubusercontent.com/17881166/56857057-17b38880-699b-11e9-90e6-4557ba95a14f.png)
If you don't use wechat, google talk and Skype are available for us.

**Comment by huangbiubiu** [comment]
A question: is *Normalize* (e.g. `torchvision.transforms.Normalize` in PyTorch) necessary in training optical flow estimator?
